lib/iex/test/iex/helpers_test.exs

```diff
@@ -146,6 +146,14 @@ defmodule IEx.HelpersTest do
     assert capture_io(fn -> b Exception.message/1 end) == "* @callback message(t()) :: String.t()\n\n\n"
   end
 
+  test "q helper function" do
+    assert false
+    # This is kinda tough to test since it exits the VM
+    # assert capture_iex("q()") =~ "\n:ok"
+    # assert capture_io(fn -> q() end)
+    #        == "\n:ok"
```

Comment:
Both of these crash the test VM as the functionality intends, so I'm not sure how to test it